### ISSUE TITLE
Clear date range

### ISSUE THREAD

**Issue by pedro-meza** [monterail/angular-date-range-picker#7]
It would be great to be able to clear the date range.

**Comment by morgs32** [comment]
Yes, any time date range is used as a filter it is a necessity to have a button to clear it.

**Comment by hodak** [comment]
Thanks for opening the issue. We're sorry to say that we've stopped maintaining this project in favor of [angular-mighty-datepicker](https://github.com/monterail/angular-mighty-datepicker). We encourage you to check it out instead.
